Security/Guidelines/OpenSSH.mediawiki

```diff
@@ -176,7 +189,7 @@ account    required     pam_nologin.so
 
 * NIST curves (<code>ecdh-sha2-nistp512,ecdh-sha2-nistp384,ecdh-sha2-nistp256</code>) are listed for compatibility, but the use of <code>curve25519</code> is [https://safecurves.cr.yp.to/ generally preferred].
 
-* SSH protocol 2 supports [https://en.wikipedia.org/wiki/Diffie%E2%80%93Hellman_key_exchange DH] and [https://en.wikipedia.org/wiki/Elliptic_curve_Diffie%E2%80%93Hellman ECDH] key-exchange as well as [https://en.wikipedia.org/wiki/Forward_secrecy forward secrecy].
+* SSH protocol 2 supports [https://en.wikipedia.org/wiki/Diffie%E2%80%93Hellman_key_exchange DH] and [https://en.wikipedia.org/wiki/Elliptic_curve_Diffie%E2%80%93Hellman ECDH] key-exchange as well as [https://en.wikipedia.org/wiki/Forward_secrecy forward secrecy]. When DH (not ECDH) is used, the moduli the server proposes should be at least 2048-bit-long and 3072-bit-long whenever possible.
```

Comment:
I could not find anything related to DH and ECDH but it should probably be added to that page rather than on several pages for maintainability. Linking anyway for now.